scripts/e2e_test.sh

```diff
@@ -233,9 +233,15 @@ readonly REQUIRED_PATTERNS=(
   'msg="calculated inputs added via f.Add()" target=FuzzParseComplex package=parser count=0'
   'msg="calculated inputs added via f.Add()" target=FuzzEvalExpr package=parser count=2'
   'msg="calculated inputs added via f.Add()" target=FuzzBuildTree package=tree count=0'
-  'msg="Worker starting fuzz target" workerID=1'
-  'msg="Worker starting fuzz target" workerID=2'
-  'msg="Worker starting fuzz target" workerID=3'
+  'msg="Crash still reproducible; keeping GitHub issue open" target=FuzzParseComplex package=parser'
+  'msg="Crash still reproducible; keeping GitHub issue open" target=FuzzUnSafeReverseString package=stringutils'
+  'msg="Crash still reproducible; keeping GitHub issue open" target=FuzzBuildTree package=tree'
```

Comment:
IIUC, these messages won't appear if the first time the test runs (i.e. no preexisting issues).

We handle that special case below already, so probably we should do the same for the reproducibility logs.